content/blog/2024/06/2024-06-20-matrix-v1.11-release.md

```diff
@@ -23,20 +23,20 @@ This post covers [MSC3916](https://github.com/matrix-org/matrix-spec-proposals/p
 
 Client and server developers are encouraged to support authenticated media as soon as they can to help server admins make the switch. Servers have a bit more work to do than clients, unfortunately, but both should have a reasonably easy time adding in the relevant support. The T&S team have made themselves available in [#matrix-client-developers:matrix.org](https://matrix.to/#/#matrix-client-developers:matrix.org) and [#matrix-homeserver-developers:matrix.org](https://matrix.to/#/#matrix-homeserver-developers:matrix.org) to help developers implement this new feature - feel free to visit the respective room for your project and ask questions.
 
-Below is some implementation guidance for developers, though is not a replacement for the [spec](https://spec.matrix.org/v1.11/client-server-api/#content-repository). Where this post and the spec differ, please prefer the spec :)
+Below is some implementation guidance for developers, though is not a replacement for the [spec](https://spec.matrix.org/v1.11/client-server-api/#content-repository). Where this post and the spec differ, please prefer the spec. :)
 
 
 ### Client implementation guidance
 
-Thankfully, supporting authenticated media should be relatively trivial for most clients. When the homeserver declares support for Matrix 1.11 through <code>[/versions](https://spec.matrix.org/v1.11/client-server-api/#get_matrixclientversions)</code>, the client should use the new <code>[/_matrix/client/v1/media/* endpoints](https://spec.matrix.org/v1.11/client-server-api/#content-repo-client-behaviour)</code>, noting that they now require authentication. The user’s access token can be supplied using the <code>Authorization</code> header. (Note that using the query string <code>?access_token</code> method is not recommended, and [deprecated](https://spec.matrix.org/v1.11/client-server-api/#using-access-tokens) in Matrix 1.11 for all endpoints.)
+Thankfully, supporting authenticated media should be relatively trivial for most clients. When the homeserver declares support for Matrix 1.11 through <code>[/versions](https://spec.matrix.org/v1.11/client-server-api/#get_matrixclientversions)</code>, the client should use the new [<code>/_matrix/client/v1/media/*<code> endpoints](https://spec.matrix.org/v1.11/client-server-api/#content-repo-client-behaviour), noting that they now require authentication. The user’s access token can be supplied using the <code>Authorization</code> header. (Note that using the query string <code>?access_token</code> method is not recommended, and [deprecated](https://spec.matrix.org/v1.11/client-server-api/#using-access-tokens) in Matrix 1.11 for all endpoints.)
```

Comment:
Shouldnt the code tag be closing? (I may be confused by mobile github diff)

Reply:
Uh yep